src/adapters/adapter.js

```diff
@@ -163,4 +181,13 @@ export default class Adapter extends Configurable {
                     {[this.name]: adapterUserId});
   }
 
+  promptUser = (data, cb) => {
+    this.bot.emitter.emit(AO.WHISPER_USER,this.name, data);
+    if (this.prompts[data.userId]) {
+      this.prompts[data.userId].push({data, cb});
```

Comment:
I am going to add an adapter parameter so that plugins can choose to whisper in response to a command only on that adapter.  we could also store a last-active property in the botusers object for the last adapter that processed a message for that user.